tensorflow/models/embedding/word2vec.py

```diff
@@ -378,7 +378,7 @@ def save_vocab(self):
     opts = self._options
     with open(os.path.join(opts.save_path, "vocab.txt"), "w") as f:
       for i in xrange(opts.vocab_size):
-        f.write("%s %d\n" % (tf.compat.as_text(opts.vocab_words[i]),
+        f.write("%s %d\n" % (tf.compat.as_text(opts.vocab_words[i]).encode('utf-8'),
```

Comment:
Also, use double quotes, instead of single quotes, to be consistent with the reset of the code.